ADR/Components/Aerodynamic_components/tests/test_Aerodynamic_surface.py

```diff
@@ -0,0 +1,25 @@
+from ADR.Components.Aerodynamic_components.Aerodynamic_surface import Aerodynamic_surface
+import numpy as npt
+
+
+def test_Aerodinamics_surface():
+    airfoil_Parameters = {"airfoil_clmax": 1.2, "airfoil1_name": "s1223", "airfoil2_name": "s1223",
```

Comment:
With a clmax of 1.2, the airfoil will not reach 20 degrees of angle of attack. See for yourself.
This is a sign (coming from the test, nice) that we should refactor the method itself, as it has problems.